### PR TITLE
Fix function tests flaps

### PR DESCRIPTION
**Pull request by azat**
Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Changelog:
- 01548_query_log_query_execution_ms (retries)
- 01675_distributed_bytes_to_delay_insert (retries)

### CODE REVIEW

File: tests/queries/0_stateless/01548_query_log_query_execution_ms.sql [hunk 1]
```diff
@@ -12,7 +12,7 @@ from system.query_log
 where
     query like '%01548_query_log_query_execution_ms%'
     and current_database = currentDatabase()
-    and query_duration_ms between 100 and 800
+    and query_duration_ms between 400 and 3000
```
Comment (akuzm) [inline tests/queries/0_stateless/01548_query_log_query_execution_ms.sql:15]: Why do you need an upper limit on the query duration? Maybe lower limit of 0.4 seconds is enough?

Comment (azat) [reply tests/queries/0_stateless/01548_query_log_query_execution_ms.sql:15]: I decided to leave the upper limit to ensure that 0.4 will not be 4 seconds (by some strange reason), but I'm fine with removing it altogether

**Comment by alexey-milovidov** [comment]
Ok. Now `01675_distributed_bytes_to_delay_insert` is flaky.

**Comment by azat** [comment]
>Ok. Now 01675_distributed_bytes_to_delay_insert is flaky.

Yes, that's why I've marked as draft, now 01675_distributed_bytes_to_delay_insert also fixed

**Comment by azat** [comment]
>[AST fuzzer (debug) — Fuzzer failed (1). See the logs.](https://clickhouse-test-reports.s3.yandex.net/23517/f1187aeb69109c88f0be978b8083080c7a843820/fuzzer_debug/report.html#fail1)

#23595

**Comment by alexey-milovidov** [comment]
@mergifyio update

**Comment by alexey-milovidov** [comment]
@Mergifyio update

**Review by alexey-milovidov — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
```
2021-04-25 03:04:54
Unable to continue: Traceback (most recent call last):
  File "sandbox/bin/executor.py", line 1033, in _execute
    self.task.on_execute()
  File "sandbox/projects/clickhouse/BaseOnCommitTask/base.py", line 304, in on_execute
    state, description, url = self.process(self.commit, repo, self.pull_request)
  File "sandbox/projects/clickhouse/ClickhouseBuilder/__init__.py", line 469, in process
    ccache_path = self._get_cache_path(commit, pull_request)
  File "sandbox/projects/clickhouse/ClickhouseBuilder/__init__.py", line 264, in _get_cache_path
    cache_data = sdk2.ResourceData(result)
  File "sandbox/sdk2/resource.py", line 203, in __init__
    super(ResourceData, self).__init__(resource, fastbone)
  File "sandbox/sdk2/internal/resource.py", line 751, in __init__
    self.__path = sdk2.Path(sdk2.Task.current.agentr.resource_sync(self.__resource.id, **kws))
  File "sandbox/agentr/client/__init__.py", line 310, in resource_sync
    return self.local_path(self("resource_sync", rid, fastbone=fastbone, restore=restore))
  File "sandbox/agentr/client/__init__.py", line 278, in __call__
    return c.wait()
  File "sandbox/common/joint/client.py", line 358, in wait
    data = self._get()
  File "sandbox/common/joint/client.py", line 405, in _get
    return self._get()
  File "sandbox/common/joint/client.py", line 425, in _get
    cls=cls_name, tb=data[5], propagate=True
ResourceNotAvailable: {da073518:9}: sandbox.agentr.errors.ResourceNotAvailable("Resource #2124877580 is not available at all sources [u'sandbox-yabs98'].",)
```

**Comment by alexey-milovidov** [comment]
A bug in testing infrastructure: resource with ccache is "found" but cannot download:

> Found ccache resource id 2124877580 from PR 23517 @ f1187aeb69109c88f0be978b8083080c7a843820 by attrs filter {'bundled': u'bundled', 'with_coverage': False, 'splitted': u'unsplitted', 'tidy': u'disable', 'package_type': u'binary', 'sanitizer': 'none', 'build_type': 'relwithdebuginfo', 'pr_number': 23517, 'compiler': u'clang-11'}

**Comment by alexey-milovidov** [comment]
SQLancer: https://github.com/sqlancer/sqlancer/pull/340
@mrigger please help merging.

**Comment by alexey-milovidov** [comment]
@azat Now this test has failed: 01641_memory_tracking_insert_optimize_long
https://clickhouse-test-reports.s3.yandex.net/0/1c472b014d814145023e41cbf437dac5a9dcb494/functional_stateless_tests_(release,_databaseordinary).html#fail1

**Comment by alexey-milovidov** [comment]
#23617

**Comment by mrigger** [comment]
> SQLancer: [sqlancer/sqlancer#340](https://github.com/sqlancer/sqlancer/pull/340)
> @mrigger please help merging.

Sorry for the delay, the PR is merged now.

**Pull request merged**